src/gallium/state_trackers/nine/device9.c

```diff
@@ -57,20 +57,38 @@
 
 #if defined(PIPE_CC_GCC) && (defined(PIPE_ARCH_X86) || defined(PIPE_ARCH_X86_64))
 
-#include <fpu_control.h>
+#include <features.h>
+
+typedef unsigned int fpu_control_t __attribute__ ((__mode__ (__HI__)));
+
```

Comment:
What does exactly do this ? What difference with just unsigned int ?

And why is #include <features.h> needed ?